modules/camera/backend/modules/camera/viewsets.py

```diff
@@ -0,0 +1,30 @@
+from .models import Image
+from .serializers import ImageSerializer, ImageUploadSerializer
+from rest_framework import viewsets
+from rest_framework.parsers import FileUploadParser
+from rest_framework.views import APIView
+from rest_framework import permissions, status
+from rest_framework.response import Response
+
+class ImageViewSet(viewsets.ModelViewSet):
+    """
+    A simple ViewSet for viewing and editing accounts.
+    """
+    queryset = Image.objects.all()
+    serializer_class = ImageSerializer
+    http_method_names = ["get"]
```

Comment:
Unused viewset

Reply:
its the same viewset being used by the listing url.